### PR TITLE
Adding additional configurations for Voron Trident, V0, and Switchwiare

### PR DESCRIPTION
**Pull request by jpapiez**
Adding additional configurations for remaining Voron printers after 2.4 was updated with additional nozzle configurations.

### CODE REVIEW

File: resources/profiles/Voron.json [hunk 1]
```diff
@@ -382,21 +382,141 @@
...
         {
             "name": "Voron 0.1 0.4 nozzle",
             "sub_path": "machine/Voron 0.1 0.4 nozzle.json"
         },
+        {
+            "name": "Voron 0.1 0.15 nozzle",
+            "sub_path": "machine/Voron 0.1 0.15 nozzle.json"
+        },
+        {
+            "name": "Voron 0.1 0.2 nozzle",
```
Comment (SoftFever) [inline resources/profiles/Voron.json:474]: Looks like these Voron 0.1  profiles were not uploaded.

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Comment by SoftFever** [comment]
@jpapiez 
Thank you for the profiles.
Can you take a look the comments?

File: resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "type": "machine",
+  "setting_id": "GM002",
+  "name": "Voron 0.1 0.4 nozzle",
```
Comment (SoftFever) [inline resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json:4]: The name is wrong.
Same for other profiles

File: resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "from": "system",
+  "instantiation": "true",
+  "inherits": "fdm_klipper_common",
+  "printer_model": "Voron 0.1",
+  "default_print_profile": "0.10mm Standard 0.25 nozzle @Voron",
+  "nozzle_diameter": [
+    "0.4"
```
Comment (SoftFever) [inline resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json:11]: the diameter is wrong.
Same for other profiles

**Review by SoftFever — CHANGES_REQUESTED** [review]
Thanks for the quick response.
It seems there are some issues still.

File: resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json (hunk 3 above)
Comment (jpapiez) [reply resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json]: Fixed

**Comment by jpapiez** [comment]
> Thanks for the quick response. It seems there are some issues still.

I believe all the issues are finally fixed.

**Review by SoftFever — APPROVED** [review]
Looks good.
Thank you

**Pull request merged**